ibm/service/cdtektonpipeline/resource_ibm_tekton_pipeline.go

```diff
@@ -0,0 +1,1192 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
```

Comment:
file name can we same as resource/datasource naming
->resource_ibm_cd_tekton_pipeline

Reply:
I can manually change them.. I believe

Reply:
yes..you need to manually change them